app-info/metainfo/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec.metainfo.xml

```diff
@@ -0,0 +1,51 @@
+  <url type="homepage">https://community.endlessos.com/</url>
+  <project_license>GPL-2.0-or-later</project_license>
+  <metadata_license>FSFAP</metadata_license>
+  <name>Endless</name>
+  <id>org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec.desktop</id>
+  <icon type="remote" width="512" height="512">https://uploads-community-endlessos-com.s3.dualstack.us-west-2.amazonaws.com/optimized/1X/8eff890b1bc1880b0077955aba6e2e76223a1365_2_512x512.jpg</icon>
+  <screenshots>
+    <screenshot type="default">
+      <image width="1358" height="1014">https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec/screenshot1.png</image>
+    </screenshot>
+    <screenshot type="default">
+      <image width="1358" height="1014">https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec/screenshot2.png</image>
+    </screenshot>
+    <screenshot type="default">
+      <image width="1358" height="1014">https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec/screenshot3.png</image>
+    </screenshot>
+    <screenshot type="default">
+      <image width="1358" height="1014">https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec/screenshot4.png</image>
+    </screenshot>
+    <screenshot type="default">
+      <image width="1358" height="1014">https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec/screenshot5.png</image>
+    </screenshot>
+    <screenshot type="default">
+      <image width="1358" height="1014">https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec/screenshot6.png</image>
+    </screenshot>
+    <screenshot type="default">
+      <image width="1358" height="1014">https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec/screenshot7.png</image>
+    </screenshot>
+    <screenshot type="default">
+      <image width="1358" height="1014">https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec/screenshot8.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
+    <category>Chat</category>
+  </categories>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-info">moderate</content_attribute>
+    <content_attribute id="social-audio">moderate</content_attribute>
+    <content_attribute id="social-contacts">intense</content_attribute>
+  </content_rating>
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+    <display_length compare="ge">small</display_length>
+  </recommends>
+  <summary>Endless Community</summary>
```

Comment:
Perhaps something like:

> A forum to discuss, get help with, and give feedback on Endless OS

The listing also looks a bit bare without a description, but that's one for another day.

Reply:
`appstreamcli validate` doesn't like that there aren't any descriptions.

Reply:
Since the updated forum XML had it, I added support in the generator for it.